.clang-format

```diff
@@ -1 +1,3 @@
 BasedOnStyle: Google
+DerivePointerAlignment: false
+PointerAlignment: Left
```

Comment:
Could you please remove the explicit `PointerAlignment` here? It seems to be the default:
```
$ clang-format --style=Google -dump-config|grep PointerAlign
DerivePointerAlignment: true
PointerAlignment: Left
```

Reply:
ok